chromium_src/third_party/blink/common/loader/network_utils.cc

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "third_party/blink/public/common/loader/network_utils.h"
+#include "base/feature_list.h"
+#include "build/build_config.h"
+#include "net/net_buildflags.h"
+#include "services/network/public/cpp/is_potentially_trustworthy.h"
+#include "third_party/blink/public/common/features.h"
```

Comment:
Nit: don't think these need to be included here.

Reply:
Fixed in https://github.com/brave/brave-core/pull/7194/commits/afe446d035334d5272de6786c651e74a9edf1e5c